README.md

```diff
@@ -50,162 +43,112 @@ source and public models in popular formats such as TensorFlow, ONNX, PaddlePadd
+  
+</details>
 
-See [How to build OpenVINO](./docs/dev/build.md) to get more information about the OpenVINO build process.
+### Documentation
 
-## How to contribute
+**User documentation**
 
-See [Contributions Welcome](https://github.com/openvinotoolkit/openvino/issues/17502) for good first issues.
+Find the latest documentation, from getting started guides to advanced topics, at [docs.openvino.ai](https://docs.openvino.ai/). User documentation contains detailed information about all OpenVINO components, providing you with essential details to create applications using OpenVINO.
 
-See [CONTRIBUTING](./CONTRIBUTING.md) for contribution details. Thank you!
+**Developer documentation**
 
-Visit [Intel DevHub Discord server](https://discord.gg/7pVRxUwdWG) if you need help or wish to talk to OpenVINO developers. You can go to the channel dedicated to Good First Issue support if you are working on a task.
+[Developer documentation](./docs/dev/index.md) on GitHub contains information about architectural decisions implemented within OpenVINO components. It offers all the essential information required for [building](./docs/dev/build.md)  and [contributing](./CONTRIBUTING.md) to OpenVINO.
 
-## Take the issue
-If you wish to be assigned to an issue please add a comment with `.take` command.
+For additional resources, read:
 
-## Get support
+* [Product Page](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html)
+* [Release Notes](https://docs.openvino.ai/2024/openvino_release_notes.html)
+* [OpenVINO Blog](https://blog.openvino.ai/) and [OpenVINO™ toolkit on Medium](https://medium.com/@openvino)
 
-Report questions, issues and suggestions, using:
+### Contribution and Support
 
-* [GitHub* Issues](https://github.com/openvinotoolkit/openvino/issues)
-* The [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on StackOverflow\*
-* [Forum](https://software.intel.com/en-us/forums/computer-vision)
+We welcome contributions of all kinds!
+
+Explore the list of [Good First Issues](https://github.com/openvinotoolkit/openvino/issues/17502), if you're looking for a place to start contributing.
+To be assigned to an issue, simply leave a comment with the `.take` command in the selected issue.
+Check out [Contribution Guidelines](./CONTRIBUTING.md) for more details.
+
+Ask questions and seek support on:
+
+* [GitHub Issues](https://github.com/openvinotoolkit/openvino/issues)
 * OpenVINO channels on the [Intel DevHub Discord server](https://discord.gg/7pVRxUwdWG)
+* the [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on Stack Overflow\*.
 
-## Additional Resources
+### License
+
+OpenVINO™ Toolkit is licensed under [Apache License Version 2.0](LICENSE).
+By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
```

Comment:
please remove License. It duplicates what we have on the right side of the OpenVINO Github page.

Reply:
is it acceptable by legal?

Reply:
Okay, let leave it. I just checked PyTorch and TensorFlow. And they have it. PyTorch Lightning - not.
Let us put license section to the very bottom